maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Handlers/SessionInfoHandler.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import FlyingFox
+import os
+import XCTest
+import Network
+
+@MainActor
```

Comment:
Do we need to do this on main thread?

Reply:
Yes, all other handlers also contain the `@MainActor`

Reply:
Thats not a great reason. The other handlers need @MainActor because they interact with XCTest.frawemork, which requires to be called on the main thread. I don't see code in the SessionInfoHandler that requires the main thread.